lib/restforce/concerns/sobject_collection_api.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require 'restforce/concerns/verbs'
+
+module Restforce
+  module Concerns
+    module SObjectCollectionAPI
+      extend Restforce::Concerns::Verbs
+
+      define_verbs :post, :patch, :delete
+
+      def collection_get(sobject_name, ids, fields)
+        raise ArgumentError, "ids are required" if Array(ids).empty?
+        raise ArgumentError, "fields are required" if Array(fields).empty?
+
+        api_get("composite/sobjects/#{sobject_name}",
+                ids: ids.join(','),
+                fields: fields.join(',')).body
+      end
+
+      def collection_delete(ids, opts = {})
+        all_or_none = opts.delete(:all_or_none) || false
```

Comment:
Using `Hash#delete` feels a bit strange when the mutated state isn't used afterwards:

```suggestion
        all_or_none = opts.fetch(:all_or_none, false)
```

Reply:
Sure, I think I had further use for opts originally, but I probably removed it. I can't remember for sure.
I'll update